lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -0,0 +1,106 @@
+import rule from '../index.mjs';
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
```

Comment:
Let's start with a simple property test:

```
a { unknown: 0; }
```

This should be accepted as the rule doesn't check properties.